### ISSUE TITLE
After installing the extension, it is not able to start

### ISSUE THREAD

**Issue by TonyTroeff** [jest-community/vscode-jest#545]
### Environment

1. `node -v`: v12.13.0
1. `npm -v`: 6.12.0
1. `npm ls jest` or `npm ls react-scripts` (if you haven’t ejected): jest@24.9.0

1. Operating system: Windows 10

### Prerequisite
- are you able to run jest test from command line? - Yes, I am.
- how do you run your tests from command line? (for example: `npm run test` or `node_modules/.bin/jest`) npm run test

### Steps to Reproduce
I simply installed your extension after creating a basic react project and received the following error:
![image](https://user-images.githubusercontent.com/33567383/71778818-3ef73680-2fbb-11ea-9345-92b3b8b7dbeb.png)

My username is 'Tony Troeff' but I see that the path of the module name is only "C:\Users\Tony" and this is the reason why such a reporter is not found, I suppose.

### Expected Behavior

After installing your extension, it should be able to start.

**Comment by boatcoder** [comment]
My only fix for this was to upgrade to 1.41 which fixed the issue.  I was getting no tracebacks, it just exited.

**Comment by Pierre-Demessence** [comment]
I have exactly the same issue with
```
Error: Could not resolve a module for a custom reporter.
  Module name: C:\Users\Pierre
    at options.reporters.reporters.map.reporterConfig (d:\Dev\DiscordBot\node_modules\jest-config\build\normalize.js:474:15)
    at Array.map (<anonymous>)
    at normalizeReporters (d:\Dev\DiscordBot\node_modules\jest-config\build\normalize.js:457:33)
    at normalize (d:\Dev\DiscordBot\node_modules\jest-config\build\normalize.js:544:5)
    at readConfig (d:\Dev\DiscordBot\node_modules\jest-config\build\index.js:163:46)
    at readConfigs (d:\Dev\DiscordBot\node_modules\jest-config\build\index.js:373:26)
    at d:\Dev\DiscordBot\node_modules\@jest\core\build\cli\index.js:155:58
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (d:\Dev\DiscordBot\node_modules\@jest\core\build\cli\index.js:108:24)
    at _next (d:\Dev\DiscordBot\node_modules\@jest\core\build\cli\index.js:128:9)
```
Where `Pierre Demessence` is my username.

I am also able to run jest on the command line.


### Environment
1. `node -v`: v10.16.3
2. `npm -v`: 6.13.6
3. `npm ls jest`: jest@24.9.0
4. Operating system: Windows 10

**Comment by connectdotz** [comment]
this looks like a bug. Does your home directory happen to contain a space? Looks like the reporter module name all got truncated after your first name...

**Comment by Pierre-Demessence** [comment]
I confirm #547 has fixed the issue after updating to 3.1.1! 
Thanks for being so quick! 🙂
